### PR TITLE
feat(flavored-markdown): add copy action to code blocks

### PR DESCRIPTION
**Pull request by emoralesb05**
## Description

Add copy code action when markdown is rendered as code blocks to allow users to easily copy and paste code examples.

#### Test Steps

- [ ] `npm run serve`
- [ ] Go to flavored-markdown demo
- [ ] See copy code action and use it

#### General Tests for Every PR

- [ ] `npm run serve:prod` still works.
- [ ] `npm run tslint` passes.
- [ ] `npm run stylelint` passes.
- [ ] `npm test` passes and code coverage is not lower.
- [ ] `npm run build:lib` still works.

### CODE REVIEW

**Comment by christianmemije** [comment]
@emoralesb05 Why not add this directly to `td-highlight` in case people want to copy code without having to use the `td-flavored-markdown` component.

**Comment by christianmemije** [comment]
Might be good to give the user some feedback that copy was successful. Maybe a quick checkbox/tooltip/etc

File: src/platform/flavored-markdown/README.md [hunk 1]
```diff
@@ -15,6 +15,9 @@ By default, `--dev` build will log the following message in the console to let y
 + content: string
   + Markdown format content to be parsed as material/covalent markup. 
   + Used to load data dynamically. e.g. `README.md` content.
++ showCopyCodeAction: string
```
Comment (jotatoledo) [inline src/platform/flavored-markdown/README.md]:
```suggestion
+ showCopyCodeAction: bool
```

**Comment by emoralesb05** [comment]
Closing this since the discussed implementation is totally different than this.

**Pull request closed without merging**